CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2010, 2011, 2012, 2013, 2014, 2018  Olga Yakovleva <yakovleva.o.v@gmail.com>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+cmake_minimum_required(VERSION 3.16)
```

Comment:
VERSION 3.16? What for?

Reply:
Because I use it. And I don't think that supporting damn legacy a good idea. I have scripts building packages with latest CMake for Debian quite fast. In fact they just download prebuilt binaries and package them.

Reply:
>Because I use it.

You need to think not only about yourself.

>And I don't think that supporting damn legacy is a good idea.

You need to think not only about yourself.

Reply:
As you wish. Downgraded to 3.5.1. Haven't tested on it though.

Reply:
CMake 3.8. Not work.

Reply:
CMake 3.8. Works.